src/main/kotlin/uk/gov/justice/digital/hmpps/approvedpremisesapi/controller/cas1/SpaceBookingController.kt

```diff
@@ -0,0 +1,46 @@
+package uk.gov.justice.digital.hmpps.approvedpremisesapi.controller.cas1
+
+import org.springframework.http.ResponseEntity
+import org.springframework.stereotype.Service
+import uk.gov.justice.digital.hmpps.approvedpremisesapi.api.cas1.SpaceBookingsCas1Delegate
+import uk.gov.justice.digital.hmpps.approvedpremisesapi.api.model.Cas1SpaceBooking
+import uk.gov.justice.digital.hmpps.approvedpremisesapi.api.model.NewCas1SpaceBooking
+import uk.gov.justice.digital.hmpps.approvedpremisesapi.service.OffenderService
+import uk.gov.justice.digital.hmpps.approvedpremisesapi.service.UserService
+import uk.gov.justice.digital.hmpps.approvedpremisesapi.service.cas1.Cas1SpaceBookingService
+import uk.gov.justice.digital.hmpps.approvedpremisesapi.transformer.cas1.Cas1SpaceBookingTransformer
+import uk.gov.justice.digital.hmpps.approvedpremisesapi.util.extractEntityFromValidatableActionResult
+import java.util.UUID
+
+@Service
+class SpaceBookingController(
+  private val userService: UserService,
+  private val offenderService: OffenderService,
+  private val spaceBookingService: Cas1SpaceBookingService,
+  private val spaceBookingTransformer: Cas1SpaceBookingTransformer,
+) : SpaceBookingsCas1Delegate {
+  override fun placementRequestsPlacementRequestIdSpaceBookingsPost(
+    placementRequestId: UUID,
+    body: NewCas1SpaceBooking,
+  ): ResponseEntity<Cas1SpaceBooking> {
+    val user = userService.getUserForRequest()
+
+    val booking = extractEntityFromValidatableActionResult(
+      spaceBookingService.createNewBooking(
+        body.premisesId,
+        placementRequestId,
+        body.arrivalDate,
+        body.departureDate,
+        user,
+      ),
+    )
+
+    val person = offenderService.getPersonInfoResult(
+      booking.placementRequest.application.crn,
+      user.deliusUsername,
+      false,
```

Comment:
I think we should be using user.hasQualification(UserQualification.LAO) to determine this boolean (this is what we do on the active 'create booking for placement request' endpoint.

Reply:
I believe this is redundant as the AP-and-Delius API should be taking this into account when presenting offender details, but I'm not 100% sure on this. Happy to change it if that's not the case.

Reply:
Okay, makes sense - change has been made.